lib/std/decode_helpers.nim

```diff
@@ -1,13 +1,11 @@
-# Include file that implements 'decodePercent' and friends. Do not import it!
-
-proc handleHexChar(c: char, x: var int, f: var bool) {.inline.} =
+proc handleHexChar*(c: char, x: var int, f: var bool) {.inline.} =
   case c
   of '0'..'9': x = (x shl 4) or (ord(c) - ord('0'))
   of 'a'..'f': x = (x shl 4) or (ord(c) - ord('a') + 10)
   of 'A'..'F': x = (x shl 4) or (ord(c) - ord('A') + 10)
   else: f = true
 
-proc decodePercent(s: string, i: var int): char =
+proc decodePercent*(s: string, i: var int): char =
   ## Converts `%xx` hexadecimal to the charracter with ordinal number `xx`.
```

Comment:
pre-existing but now that it's exported, it needs some cleanup s/charracter/character/